### DISCUSSION TITLE
archives/wiki/debian-%E6%8C%82%E8%BD%BD%E5%AE%98%E6%96%B9-iso-%E4%B8%BA%E6%9C%AC%E5%9C%B0-apt-%E8%BD%AF%E4%BB%B6%E6%BA%90/

### DISCUSSION THREAD

**Discussion by giscus** [songtianlun/songtianlun.github.io#16] [Announcements]
# archives/wiki/debian-%E6%8C%82%E8%BD%BD%E5%AE%98%E6%96%B9-iso-%E4%B8%BA%E6%9C%AC%E5%9C%B0-apt-%E8%BD%AF%E4%BB%B6%E6%BA%90/

debian 挂载官方 iso 为本地 apt 软件源 首先，下载debian 10.1.0 amd64架构的dvd iso，一般来说第一张就包含了大部份需要用到的软件，官方提供共计四张

https://www.frytea.com/archives/wiki/debian-%E6%8C%82%E8%BD%BD%E5%AE%98%E6%96%B9-iso-%E4%B8%BA%E6%9C%AC%E5%9C%B0-apt-%E8%BD%AF%E4%BB%B6%E6%BA%90/

**Comment by Paper-Dragon** [comment]
挺好，刚好需要这个
